markerclustererplus/src/markerclusterer.js

```diff
@@ -286,6 +298,10 @@ ClusterIcon.prototype.show = function () {
       this.div_.title = this.sums_.title;
     }
     this.div_.style.display = "";
+
+    this.syncDivStyle({
+      title: this.div_.title,
+    });
```

Comment:
Don't you have to synch the display style as well?

Reply:
My intention was only to set the props that the code sets on the visible had set. Line 388 copies all of the css set on `this.dev_`